src/libraries/System.Runtime/tests/TrimmingTests/DefaultValueAttributeCtorTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reflection;
+
+class Program
+{
+    static int Main(string[] args)
+    {
+        var attribute = new DefaultValueAttribute(typeof(string), "Hello, world!");
+
+        // There's a fallback in the DefaultValueAttribute ctor for when the following
+        // value is null. It shouldn't be null at this point, if the linker didn't trim
+        // System.ComponentModel.TypeConverter.ConvertFromInvariantString out.
+        object convertFromInvariantString = typeof(DefaultValueAttribute).GetField(
+            "s_convertFromInvariantString",
+            BindingFlags.GetField | BindingFlags.NonPublic | BindingFlags.Static).GetValue(attribute);
```

Comment:
Maybe add a comment into the product code that the field is accessed via private reflection from test - I know the test should catch any changes to the field... I don't know if we have a pattern to handle this kind of testing in libraries.